packages/instant/src/util/asset_data.ts

```diff
@@ -0,0 +1,18 @@
+import { AssetProxyId } from '@0xproject/types';
+
+import { assetMetaData } from '../data/asset_meta_data';
+
+// TODO: tests for this
+export const bestNameForAsset = (assetData: string | undefined, defaultString: string) => {
+    if (assetData === undefined) {
```

Comment:
ah I noticed that, but I try to avoid importing a library function if it's simple, easy and readable to perfrom the same thing without it, a la https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore

Reply:
Yea I kind of agree? Don't feel too strongly. However, as mentioned previously, I am a big fan of consistency and reducing decisions. It's nice to not have to think about "how am I going to check for undefined". And it's also nice for the entire codebase to do it the same way. Idk, I think since we already have lodash in the bundle it's worth doing that, but don't feel strongly about it.

Reply:
>  I think since we already have lodash in the bundle it's worth doing that

If we are able to get our bundle optimized, the bundle should only include the lodash functions that we use, and it'd be nice to be pulling in one less function if we consistently used this approach for checking for `undefined`

That being said, I hear you RE: consistency and reducing decisions.  For the sake of those two points, I'm down to use the lodash function for now. We can revisit our approach to lodash in general as a team later, and perhaps establish a new consistent pattern moving forward at that point.